# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fluxbox/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fluxbox-1.0.0/src/main.cc.vanilla	2008-07-13 15:39:31.000000000 +0200
+++ fluxbox-1.0.0/src/main.cc	2008-07-13 15:39:42.000000000 +0200
@@ -25,6 +25,7 @@
 #include "fluxbox.hh"
 #include "version.h"
 #include "defaults.hh"
+#include "string.h"
 
 #include "FbTk/Theme.hh"
 #include "FbTk/I18n.hh"
